docs/reference/module-options.md

```diff
@@ -372,3 +372,457 @@ module.exports = {
+    minimumPark: [
+      {
+        slug: '/',
+        parkedId: 'home',
+        _defaults: {
+          title: 'Welcome',  // 👈
+          type: 'welcome-page' // 👈
+        },
+        _children: [
+            {
+            slug: '/trash',
+            parkedId: 'trash',
+            type: '@apostrophecms/trash',
+            trash: true,
+            orphan: true,
+            _defaults: { title: 'Trash' }
+          }
+        ]
+      }
+    ]
+  },
+  // ...
+}
+```
+
+### `park`
+
+Use the `park` option to add an array of pages that should be created when the app starts up if they do not already exist. Each page is added as an object with initial properties, including the required `parkedId`.
+
+Required and recommended parked page properties include:
+
+| Setting | Requirement | Description |
+|---------|-------------|-------------|
+| `parkedId` | Required | A unique ID value used to identify it among parked pages. |
+| `slug` | Required | The page [slug](/reference/glossary.md#slug). |
+| `type` | Required | The page type to be used for the parked page. |
+| `title` | Recommended | The page title. If not set, it will be "New Page." |
+
+If added on the top level of the page object, these properties will not be editable through the user interface. Properties other than `parkedId` may be included in a `_defaults` property instead, which will allow them to be edited in the UI.
+
+#### Example
+
+```javascript
+// modules/@apostrophecms/page/index.js
+module.exports = {
+  options: {
+    park: [
+      // The blog page has a permanent slug, title, and type.
+      {
+        parkedId: 'blogParkedId',
+        slug: '/blog',
+        title: 'Blog',
+        type: 'blog-page'
+      },
+      // The team page has a permanent type, but editable slug and title.
+      {
+        parkedId: 'teamParkedId',
+        type: 'staff-page',
+        _defaults: {
+          slug: '/team',
+          title: 'Our Team',
+        }
+      }
+    ]
+  },
+  // ...
+}
+```
+
+### `publicApiProjection` (for pages)
+
+By default the built-in Apostrophe REST APIs are not accessible without proper [authentication](/reference/api/authentication.md). You can set an exception to this for `GET` requests to return specific document properties with the `publicApiProjection` option.
+
+This should be set to an object containing individual field name keys set to `1` for their values. Those fields names included in the `publicApiProjection` object will be returned when the `GET` API requests are made without authentication.
+
+#### Example
+
+```javascript
+// modules/@apostrophecms/page/index.js
+module.exports = {
+  options: {
+    publicApiProjection: {
+      title: 1,
+      _url: 1 // 👈 Dynamic properties are allowed
+    }
+  },
+  // ...
+}
+```
+
+Unauthenticated [`GET /api/v1/@apostrophecms/page`](api/pages.md#get-api-v1-apostrophecms-page) requests would return each piece with only the `title` and `_url` properties.
+
+### `quickCreate` (for pages)
```

Comment:
I'm assuming we cover quickCreate elsewhere too

Reply:
Yeah, for pieces as well. But both are not included in "doc-type"